experimental/devices/sn3218/sn3218.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package sn3218
+
+import (
+	"errors"
+
+	"periph.io/x/periph/conn/i2c"
+)
+
+const (
+	i2cAddress             = 0x54
+	cmdEnableOutput        = 0x00
+	cmdSetBrightnessValues = 0x01
+	cmdEnableLeds          = 0x13
+	cmdUpdate              = 0x16
+	cmdReset               = 0x17
+)
+
+// Dev is a handler to sn3218 controller.
+type Dev struct {
+	i2c        i2c.Dev
+	states     [18]bool
+	brightness [18]byte
+}
+
+// New returns a handle to a SN3218 LED driver.
+func New(bus i2c.Bus) (Dev, error) {
+	d := i2c.Dev{Bus: bus, Addr: i2cAddress}
+	dev := Dev{}
+	dev.i2c = d
+	dev.reset()
+	return dev, nil
+}
+
+// Halt resets the registers and switches the driver off.
+func (d *Dev) Halt() error {
+	if err := d.Disable(); err != nil {
```

Comment:
This is not necessary, reset() puts the device in power save mode.

Reply:
agree, fixed